docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -4,13 +4,26 @@ title:  Add a new element to the Marketplace
 sidebar_label: Add a new element to the Marketplace
 ---
 
-:::info
-If you want to contribute to Mia-Platform Marketplace with a new component, you can [open an issue](https://github.com/mia-platform/community/issues/new?labels=marketplace&template=marketplace-contribution.md&title=Add+new+marketplace+item), wherein you will find what information is needed for the request.
+In this section, we dive into the process of creating a new component for the Marketplace, analyzing the main differences between different types of components and providing some configuration examples.
+
```

Comment:
Let's add a note that marketplace items can only be created by people with access to the Mia-Platform Console CMS, therefore you may have to ask your Mia-Platform referent to add your marketplace items for you.